pkg/service/service.go

```diff
@@ -464,6 +466,21 @@ func (s *Service) upsertServiceIntoLBMaps(svc *svcInfo, prevBackendCount int,
 		}
 	}
 
+	// Trigger GC if necessary for purging stale CT_SERVICE entries
+	if option.Config.EnableHostServicesTCP == false ||
+		option.Config.EnableHostServicesUDP == false ||
+		option.Config.EnableNodePort == true &&
+			svcType != lb.SVCTypeClusterIP {
+		sig := signal.SignalMsg{
+			Which: signal.SignalGC,
+		}
+		if ipv6 {
+			sig.Data = signal.SignalGCV6
+		} else {
+			sig.Data = signal.SignalGCV4
+		}
+		signal.SignalTrigger(&sig, "upsert_service")
```

Comment:
Are these coalesced somehow? Will a burst of services or update trigger this many times in a short window?